src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1139,6 +1136,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 OutputFileName = outputFileName,
                 PdbPath = pdbPath,
                 EmitPdb = emitPdb,
+                // TODO: pipe debugInformationFormat through this
```

Comment:
Oops.  I added that back before I learned that the format was already passed through on line 1116.  It should be removed; there's no more work left.